src/filepath.c

```diff
@@ -1358,17 +1358,26 @@ f_mkdir(typval_T *argvars, typval_T *rettv)
 f_pathshorten(typval_T *argvars, typval_T *rettv)
 {
     char_u	*p;
+    int		trim_len;
+
+    if (argvars[1].v_type != VAR_UNKNOWN) {
+	trim_len = (int)tv_get_number(&argvars[1]);
+	if (trim_len < 1) trim_len = 1;
+    } else {
```

Comment:
In Vim code:
*  curly braces go on a separate line.
* Indentation should be 4 spaces